update

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash
+
+set -eu -o pipefail
+cd "$(dirname "${BASH_SOURCE[0]}")"
+source $source_terraform_hercules_ci/nix/update-lib.sh
```

Comment:
This seems quite fragile, if something changes upstream, it's going to break here. I'm not confident in bash enough to know when this will happen.

Reply:
I agree, but it's pinned and we control upstream. We'll have multiple example repos and I'm not looking forward to maintaining duplicated scripts manually.

Reply:
I'll pin it separately.